source/openapi-spec.yml

```diff
@@ -357,3 +400,16 @@ components:
           type: string
           description: A brief written description of the work involved
           example: "I lead, develop and enhance the literacy teaching practice of others..."
+    Error:
```

Comment:
Would be useful to have an example for each property when looking at it under Schemas.

Reply:
I was thinking that the Error schema is meant to be generic across all error responses. 
I think that having examples in the actual response rather then the Error schema which is more there for the "shape".